docker-compose-rule-core/src/test/java/com/palantir/docker/compose/execution/DockerComposeShould.java

```diff
@@ -193,15 +193,15 @@ public void pass_concatenated_arguments_to_executor_on_docker_compose_run()
 
     @Test
     public void return_the_output_from_the_executed_process_on_docker_compose_exec() throws Exception {
-        String lsString = "-rw-r--r--  1 user  1318458867  11326 Mar  9 17:47 LICENSE\n"
-                + "-rw-r--r--  1 user  1318458867  12570 May 12 14:51 README.md";
+        String lsString = String.format("-rw-r--r--  1 user  1318458867  11326 Mar  9 17:47 LICENSE%n"
```

Comment:
There are indeed a lot of "\n" in test strings, but I think this is the only place that matters. Command::processOutputFrom(Process) reads the lines from `process.getInputStream()` and joins them using `System.lineSeparator()`, so lsString's "\n" gets read as "\r\n" on Windows, hence the problem here. The other places where "\n" is used (e.g. ContainerNameShould) just test that the linux line endings work. I added `ContainerNameShould.allow_windows_newline_characters` to test the windows line endings.